gammapy/data/tests/test_observationlist.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.coordinates import SkyCoord
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_data
+from ...utils.energy import EnergyBounds
+from ...data import DataStore, ObservationList
+
+
+@requires_data('gammapy-extra')
+def test_make_psftable():
```

Comment:
@cdeil 
hi,
 I took two observations of the Crab in gammapy extra: 23523 and 23526 for testinf the mean psf computation.
The problem is that for these two observations the psfvalue you obtain for the EnergydependantTablePsf (```psf1``` and ```psf2```) are really different so this is weird knowing that the Crab offset in the FOV for these two observations is almost equal (0.49 and 0.51 degree…). I think that there is either a problem on the interpolation of the parameters of the tripple gauss for these runs in your exporter or in the definition of the tripple gauss in gammapy. The difference between the two psf is superior to 10 order of magnitude so there is obviously a bug... 

When I am doing the same test for the two same observations using the PA data and their king profile for the psf, it works fine. The mean psf value integrated between 1 and 10 TeV of the two runs is well between the values of each run and the R68 of the psf of each observation and of the psfmean is almost equal.

So I think what I coded is ok but for the moment I can’t do any test with the data of HAP in gammapy-extra …